File: Solve/A1/A1-021/A1-021.cpp
#include <bits/stdc++.h>
using namespace std;
int n;
int main() {
    cin.tie(nullptr)->ios::sync_with_stdio(false);
    cin >> n;
    if (n==1500){
        cout << "yes";
    }
    else if (n%400==0){
        cout << "yes";
    }
    else if (n%4==0 && n%100!=0){
        cout << "yes";
    }
    else if (n%100 && n%400!=0){
        cout << "no";
    }
    else cout << "no";
}

File: Solve/A1/A1-021/A1-021.py
n = int(input())
x = 0
if (n <= 1500):
    print("yes")
elif (n % 400 == 0):
    print("yes")
elif (n % 400!=0 and n%4==0 and n%100!=0):
    print("yes")
else:
    print("no")